services/rest/src/identities.ts

```diff
@@ -11,25 +11,37 @@ const identityDb = new IdentityModel(STAGE);
 const queryBy = {
   address: (str: string) => identityDb.getIdentityByAddress(str),
   username: (str: string) => identityDb.getIdentityByUsername(str),
+  email: (str: string) => identityDb.getIdentityByEmail(str),
 };
 
+declare type FilterKey = 'address' | 'email' | 'username';
```

Comment:
```suggestion
type FilterKey = 'address' | 'email' | 'username';
```

Reply:
Fixed in [f2019f3](https://github.com/FleekHQ/space-services/pull/36/commits/f2019f36d1bead6cd42ef229f113d6b9c4a288ab).